### PR TITLE
10.0 mig base continent

### PR DESCRIPTION
**Pull request by qtheuret**
As I can't re-open the pull request #532, I create this new one for the migration of the base_continent module to v10.0.

Sorry for the inconvenience with my multiple pull requests but I'm a newbie with Git :)

### CODE REVIEW

File: base_continent/README.rst
Comment (JayVora-SerpentCS) [inline base_continent/README.rst]: https://runbot.odoo-community.org/runbot/134/10.0

File: base_continent/__manifest__.py
Comment (yajo) [inline base_continent/__manifest__.py:11]: No need for this if you depend on other thing

**Review by yajo — APPROVED** [review]

File: base_continent/__manifest__.py
Comment (yajo) [reply base_continent/__manifest__.py:11]: otoh, why do you depend on sales team now?

Comment (qtheuret) [reply base_continent/__manifest__.py:11]: Because some menus (Localisation)  are moved to sales_team menu.

File: base_continent/data/continent_data.xml
Comment (NL66278) [inline base_continent/data/continent_data.xml:4]: ```<data>``` is deprecated. Put the noupdate directly on the odoo element. Remove data element and move the indentation of the rest.

File: base_continent/data/country_data.xml
Comment (NL66278) [inline base_continent/data/country_data.xml:3]: Same here, data element deprecated

File: base_continent/__manifest__.py
Comment (NL66278) [inline base_continent/__manifest__.py:2]: © is supposed to be replaced by the full word Copyright according to latest conventions (remark for all occurrences in all files)

**Review by TelmoSenseFly — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash commits but logical set/author before merging using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by pedrobaeza** [comment]
I'm afraid you haven't rebased properly. What I do is (having `origin` remote the OCA one):
```
git fetch origin
git rebase -i origin/11.0
<make whatever I want with the commits>
<save>
```

File: base_continent/__manifest__.py
Comment (legalsylvain) [reply base_continent/__manifest__.py:11]: We (with @alexis-via) had the same discussion, and decided to move into technical settings. So you don't force to install an undesired module.
ref : https://github.com/OCA/l10n-france/pull/131#issuecomment-364285722

**Review by legalsylvain — CHANGES_REQUESTED** [review]
I think we should keep dependency to base only. Otherwise, LGTM. Thanks for the work.

could you add a link, in the readme to mention what is the norm the code you introduced ? 

regards.

File: base_continent/models/country.py
Comment (huyly0909) [inline base_continent/models/country.py:7]: Please check the import order https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: base_continent/models/partner.py
Comment (huyly0909) [inline base_continent/models/partner.py:7]: Please check the import order https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: base_continent/models/continent.py
Comment (huyly0909) [inline base_continent/models/continent.py:7]: Please check the import order https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

**Comment by NL66278** [comment]
Despite failing to address some of the convention related comments, the module itself is quite usefull and not breaking anything, so I will merge nevertheless.

**Pull request merged**
